content/docs/reference/routes/from.md

```diff
@@ -10,16 +10,72 @@ pagination_next: null
 
 # From
 
-- `yaml`/`json` setting: `from`
-- Type: `URL` (must contain a scheme and hostname, must not contain a path)
-- Schemes: `https`, `tcp+https`
-- Required
-- Example: `https://verify.corp.example.com`, `tcp+https://ssh.corp.example.com:22`
+---
+
+**YAML/JSON setting:** `from` <br/> **Type:** `URL` (must contain a scheme and hostname, must not contain a path) <br/> **Schemes:** `https`, `tcp+https` <br/> **Required** <br/>
+
+**Example:**
+
+```yaml
+routes:
+  - from: https://verify.corp.example.com
+  - to: https://example.com
+
+# Or
+
+routes:
+  - from: tcp+https://ssh.corp.example.com:22
+  - to: tcp://example.com:22
+```
+
+---
 
 `From` is the externally accessible URL for the proxied request.
 
 Specifying `tcp+https` for the scheme enables [TCP proxying](/docs/capabilities/tcp/) support for the route. You may map more than one port through the same hostname by specifying a different `:port` in the URL.
 
+## Wildcard From Routes
+
+**Kubernetes:** Not supported <br/>
```

Comment:
This is unofficially supported but the behavior of the two implementations are different. The behavior will be different than what is defined in the kubernetes docs. We should warn users that the behavior may be undefined. 

\cc @wasaga to describe distinction.